### DISCUSSION TITLE
How can I report issues with the documentation?

### DISCUSSION THREAD

**Discussion by wolffvel** [jeffreylanters/react-unity-webgl#464] [Question and Answer]
I noticed that the Dynamic Device Pixel Ratio example in the docs (https://react-unity-webgl.dev/docs/advanced-examples/dynamic-device-pixel-ratio) is incomplete. The example imports useEffect from React and declares a handleChangePixelRatio function but doesn't use either of them.

**Comment by jeffreylanters** [comment]
Hi! You can open a pull request on this repository. Since it's a mono-repo, the documentation is located here as well. Thanks for your contribution!
